recipes/sqlite3/all/CMakeLists.txt

```diff
@@ -124,18 +124,16 @@ endif()
 
 if(THREADSAFE)
     find_package(Threads REQUIRED)
-    target_link_libraries(${PROJECT_NAME}  PRIVATE Threads::Threads)
+    target_link_libraries(${PROJECT_NAME} PRIVATE Threads::Threads)
 endif()
 
 if(NOT OMIT_LOAD_EXTENSION)
-    target_link_libraries(${PROJECT_NAME}  PRIVATE ${CMAKE_DL_LIBS})
+    target_link_libraries(${PROJECT_NAME} PRIVATE ${CMAKE_DL_LIBS})
 endif()
 
 if(ENABLE_FTS5 OR ENABLE_MATH_FUNCTIONS)
-    find_library(MATH_LIBRARY m)
-    if(MATH_LIBRARY)
-        target_link_libraries(${PROJECT_NAME}  PRIVATE ${MATH_LIBRARY})
-    endif()
+    find_library(MATH_LIBRARY m REQUIRED)
+    target_link_libraries(${PROJECT_NAME} PRIVATE ${MATH_LIBRARY})
```

Comment:
And what happen on platforms where math implementation is in libc or vc runtime?

Reply:
I've only tested it n Linux :(